SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h

```diff
@@ -162,6 +162,7 @@ const Worker_ComponentId GDK_KNOWN_ENTITY_TAG_COMPONENT_ID = 2007;
 const Worker_ComponentId TOMBSTONE_TAG_COMPONENT_ID = 2008;
 const Worker_ComponentId ROUTINGWORKER_TAG_COMPONENT_ID = 2009;
 const Worker_ComponentId LAST_EC_COMPONENT_ID = 2010;
+const Worker_ComponentId STRATEGYWORKER_TAG_COMPONENT_ID = 2011;
```

Comment:
To confirm, but I think a new tag is not needed. The strategy worker should use LB_TAG_COMPONENT_ID